.circleci/config.yml

```diff
@@ -63,6 +64,7 @@ jobs:
           name: binaries
           command: |
             export PATH="$GOBIN:$PATH"
+            make go-mod-cache
```

Comment:
I'm also not a circcleci expert but line 62 reads `- *dependencies`.  And the dependencies anchor is defined by:
```yaml
deps: &dependencies
  run:
    name: dependencies
    command: |
      export PATH="$GOBIN:$PATH"
      make go-mod-cache
```
Shouldn't `make go-mod-cache` already have been called there?

Reply:
Removed